test/quantization/fx/test_quantize_fx.py

```diff
@@ -4771,6 +4771,41 @@ def _test(prepare_fn, qconfig_dict):
         _test(prepare_fx, get_default_qconfig_mapping())
         _test(prepare_qat_fx, get_default_qat_qconfig_mapping())
 
+    def test_channel_shuffle_lowering(self):
+        # Three versions of channel shuffle
+        class M1(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.op = torch.nn.ChannelShuffle(2)
+
+            def forward(self, x):
+                return self.op(x + x) + x
+
+        class M2(torch.nn.Module):
+            def forward(self, x):
+                return torch.channel_shuffle(x + x, 2) + x
+
+        class M3(torch.nn.Module):
+            def forward(self, x):
+                return torch.nn.functional.channel_shuffle(x + x, 2) + x
+
+        x = torch.randn(4, 4, 4, 4)
+        for m in (M1().eval(), M2().eval(), M3().eval()):
+            m = prepare_fx(m, {"": default_qconfig}, example_inputs=(x,))
+            m_copy = copy.deepcopy(m)
+            m = convert_fx(m)
+            m_ref = convert_to_reference_fx(m_copy)
+            node_occurrence = {
+                ns.call_function(torch.quantize_per_tensor): 1,
```

Comment:
do we want to check that the op is quantized as well?

Reply:
in this case quantized just means we use the original op, but it would still be goo to explicitly test this

Reply:
> do we want to check that the op is quantized as well?

OK. Now we check occurrences of the op itself. Do you mean that?

Reply:
Looks like `torch.channel_shuffle` and `torch.nn.functional.channel_shuffle` are actually the same thing. So, we only check occurrence of either `torch.channel_shuffle` or `torch.nn.functional.channel_shuffle`.

Reply:
I didn't see the expected node appear here, should we add that?

Reply:
> I didn't see the expected node appear here, should we add that?

Do you mean line 5329?

Reply:
oh I see, sorry I missed that, LGTM